### PR TITLE
Selenium:  Finish creating the test methods in the 'UploadIntoProjectTest'

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Add the test methods to the selenium test 'UploadIntoProjectTest'
* Finish creating the test methods in the ''UploadIntoProjectTest'

### What issues does this PR fix or reference?
#10204

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/UploadIntoProjectTest.java [hunk 1]
```diff
@@ -78,37 +79,224 @@ public void selectProjectItem() {
...
+    projectExplorer.waitVisibleItem(format("%s/%s", PROJECT_NAME, uploadingFileName));
+
+    // Check that uploading file doesn't overwrite existed one
+    // when change the file
+    projectExplorer.openItemByPath(pathToUploadingFileInsideTheProject);
+    editor.typeTextIntoEditor(TEXT_TO_INSERT);
+
+    // when re-upload the file
+    projectExplorer.waitAndSelectItem(PROJECT_NAME);
+
```
Comment (Ohrimenko1988) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/UploadIntoProjectTest.java]: if steps belong to one logical part "// when re-upload the file" better do not separate they by empty rows

**Review by Ohrimenko1988 — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/UploadIntoProjectTest.java (hunk 1 above)
Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/UploadIntoProjectTest.java]: You are right, I'll correct

**Comment by artaleks9** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/496/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/496//Selenium_tests_report)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10475)
docker image: eclipseche/che-server:10475
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by artaleks9** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/498/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/498//Selenium_tests_report)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10475)
docker image: eclipseche/che-server:10475
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by artaleks9** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/500/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/500//Selenium_tests_report)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10475)
docker image: eclipseche/che-server:10475
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by artaleks9** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/504/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/504//Selenium_tests_report)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10475)
docker image: eclipseche/che-server:10475
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**